channels/whatsapp/index.rst

```diff
@@ -0,0 +1,247 @@
+   displayed once!
+
+   .. figure:: /images/channels/whatsapp/created-system-user.png
+      :alt: Get your app token from the system user
+      :align: center
+
+
+   Choose your created app, a expiration date of the token (we recommend
+   "Never") and the permissions ``whatsapp_business_management`` and
+   ``whatsapp_business_messaging``:
+
+   .. figure:: /images/channels/whatsapp/create-app-token.png
+      :alt: Create your app token
+      :align: center
+      :scale: 60%
+
+App Secret
+   Enter your app secret. You can find it in your
+   `app dashboard <https://developers.facebook.com/apps/>`_ under "App settings
+   > Basic" by clicking on the "Show" button:
+
+   .. figure:: /images/channels/whatsapp/app-secret.png
+      :alt: App dashboard with app secret
+      :scale: 60%
+
+
+After entering your data, click on next. In the background, Zammad tries
+to fetch your assigned phone number, which you can select in the next step.
+
+Step 2
+^^^^^^
+
+The second step is about your phone number and messaging details:
+
+.. figure:: /images/channels/whatsapp/setup-step-2.png
+   :alt: Step 2 of adding a WhatsApp account
+   :align: center
+   :scale: 80%
+
+Phone Number
+   Choose the phone number you want to use for the channel. Zammad should have
+   fetched the numbers which are assigned to your business app. If not, your
+   credentials from step 1 may be wrong.
+
+Welcome Message
+   Here you can define a message which is displayed to your customers
+   when they are writing to you.
+
+Goodbye Message
+   Here you can define a message which is displayed when the conversation ends.
+
+Target Group
+   Here you can define the :doc:`group </manage/groups/index>` in which the
+   ticket should be created.
+
+After entering your data, click on submit. After this step, the WhatsApp channel
+is technically created. However, there is another important step to take.
+
+Step 3
+^^^^^^
+
+This third step is about telling WhatsApp, where they should send their data:
+
+.. warning:: If you don't follow the instructions properly, your
+   account shows up but you aren't able to receive messages (because
+   Meta/WhatsApp doesn't know where to send them)!
+
+.. figure:: /images/channels/whatsapp/setup-step-3.png
+   :alt: Step 3 of adding a WhatsApp account
+   :align: center
+   :scale: 80%
+
+Callback URL
+   Click on the copy icon on the right side and head over to your WhatsApp
+   business app. Enter the copied URL in the corresponding field in your app.
+   You can find it in your
+   `"App Dashboard" <https://developers.facebook.com/apps/>`_ under
+   "WhatsApp > Configuration > Webhook > Callback URL" (see screenshot below).
+   Paste it there.
+
+Verify Token
+   Copy this string too. Enter it directly below the pasted Callback URL.
+
+.. figure:: /images/channels/whatsapp/app-dashboard-configuration-webhook.png
+   :alt: Configuration section in WhatsApp app dashboard
+
```

Comment:
We are missing here a click on the `Verify webhook` button on the WhatsApp side. This step is important, as it may fail if the Zammad system is not reachable at this point! Perhaps add a note on this.